src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,47 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+  constructor(name) {
+    this.health = 100;
```

Comment:
```suggestion
  constructor(name, health = DEFAULT_HEALTH) {
    this.health = health;
```
- You should allow to set different health as well, 100 is only by default in case it's not passed
- And 100 is a magic valu, so create a constant